pennylane/qchem/structure.py

```diff
@@ -227,10 +229,12 @@ def excitations(electrons, orbitals, delta_sz=0):
             ``sz[p] + sz[p] - sz[r] - sz[s] = delta_sz`` for the spin-projection ``sz`` of
             the orbitals involved in the single and double excitations, respectively.
             ``delta_sz`` can take the values :math:`0`, :math:`\pm 1` and :math:`\pm 2`.
+        fermionic (bool): Return a list of ``FermiWord`` objects instead of a list of lists
 
     Returns:
         tuple(list, list): lists with the indices of the spin orbitals involved in the
-        single and double excitations
+        single and double excitations. By default the lists contain lists, otherwise
```

Comment:
Please consider updating to this or something similar:
"By default the the lists contain integers representing the orbitals ..."